src/main/java/com/simibubi/create/content/contraptions/base/KineticTileEntityRenderer.java

```diff
@@ -38,10 +38,10 @@ protected void renderSafe(KineticTileEntity te, float partialTicks, PoseStack ms
 		int light, int overlay) {
 		if (Backend.getInstance().canUseInstancing(te.getLevel())) return;
 
-		// TODO
```

Comment:
There is no point in changing this because it creates the same outcome but in a less efficient way. The current code is working as well as the commented code, and I put the TODO here to look at fixing it completely when multi-layer model rendering is fixed (a painful process). However, when I looked into this more on the Forge version earlier today, I found that it will not work correctly there either under certain circumstances, so it will be changed upstream in the near future. For now, I would recommend removing all of the changes to this file.
If these changes do actually make a difference, let me know, but I cannot see how that would happen.